snuba/snuba_migrations/outcomes/0006_outcomes_add_size_col.py

```diff
@@ -0,0 +1,69 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Adds back (again) the size, bytes_received columns to match schema in SaaS
+    """
+
+    blocking = True
```

Comment:
Shouldn't this be a non-blocking migration since it is just adding columns? Same comment about the blocking = True in the other migration

Reply:
fixed